src/components/VLogoLoader/VLogoLoader.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <svg
+    viewBox="0 0 34 32"
+    xmlns="http://www.w3.org/2000/svg"
+    :class="{
+      [$style.loading]: status === 'loading' && !prefersReducedMotion,
+      [$style.quiet]: status === 'loading' && prefersReducedMotion,
+    }"
+    aria-hidden="true"
+    :data-prefers-reduced-motion="prefersReducedMotion"
+    data-testid="logo-loader"
+    class="hover:bg-yellow w-16 h-16 p-4 rounded inline-flex justify-center items-center"
+  >
+    <path
+      data-logo-part-1
+      d="M0 6.962c0 3.831 3.09 6.962 6.916 6.962V0C3.09 0 0 3.111 0 6.962Z"
+    />
+    <path
+      data-logo-part-2
+      d="M10.084 6.962c0 3.831 3.091 6.962 6.916 6.962V0c-3.806 0-6.916 3.111-6.916 6.962Z"
+    />
+    <path
+      data-logo-part-3
+      d="M27.084 13.924c3.82 0 6.916-3.117 6.916-6.962C34 3.117 30.904 0 27.084 0c-3.82 0-6.916 3.117-6.916 6.962 0 3.845 3.097 6.962 6.916 6.962Z"
+    />
+    <path
+      data-logo-part-4
+      d="M0 24.153c0 3.85 3.09 6.962 6.916 6.962V17.21C3.09 17.21 0 20.322 0 24.153Z"
+    />
+    <path
+      data-logo-part-5
+      d="M10.084 24.095c0 3.83 3.091 6.962 6.916 6.962V17.152c-3.806 0-6.916 3.112-6.916 6.943Z"
+    />
+    <path
+      data-logo-part-6
+      d="M27.084 31.057c3.82 0 6.916-3.117 6.916-6.962 0-3.845-3.096-6.962-6.916-6.962-3.82 0-6.916 3.117-6.916 6.962 0 3.845 3.097 6.962 6.916 6.962Z"
+    />
+  </svg>
+</template>
+
+<script>
+import { defineComponent } from '@nuxtjs/composition-api'
+import { useReducedMotion } from '~/composables/use-media-query'
+
+import { propTypes } from './VLogoLoader.types'
+
+export default defineComponent({
+  name: 'VLogoLoader',
+  props: propTypes,
+  /**
+   * @param {import('./VLogoLoader.types').Props} props
+   * @param {import('@nuxtjs/composition-api').SetupContext} context
+   */
+  setup() {
+    const defaultWindow = typeof window !== 'undefined' ? window : undefined
```

Comment:
The default window is being used in several places now to ensure that composables work correctly with SSR. Is there a way to extract this code somehow?

Reply:
I was thinking it could be the default value in the hook, at least. Not sure if it'll work, I'll test

Reply:
I don't think you can use a default value as it'll try to evaluate `window` even when it's undefined and throw an error.